float32 kp_v # 循迹控制中的反馈通道的比例系数
float32 kp_kapa # 循迹控制中的曲率前馈通道的比例系数
float32 b0_spdcrl #车速控制中的控制输入增益
float32 wc_spdcrl #车速控制中的控制器带宽
